libbpf-tools/opensnoop.h

```diff
@@ -18,6 +19,7 @@ struct event {
 	uid_t uid;
 	int ret;
 	int flags;
+	__u16 mode;
```

Comment:
This should be added to the end of the struct. Otherwise it introduces some hole inside the struct.

Reply:
I think `rwx` would be better.